source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,120 @@
+title: Read the local master key from a file
```

Comment:
nit: Let's use title case for step titles.

See [Enforce a Document Schema](https://docs.mongodb.com/stitch/mongodb/enforce-a-document-schema/) for an example

Reply:
Will do, thanks for the example.